modules/acknowledgements/jsx/acknowledgementsIndex.js

```diff
@@ -0,0 +1,394 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import swal from 'sweetalert2';
+import Modal from 'Modal';
+import Panel from 'Panel';
+import Loader from 'Loader';
+import FilterableDataTable from 'FilterableDataTable';
+
+/**
+ * Acknowledgements Module page.
+ *
+ * Serves as an entry-point to the module, rendering the whole React
+ * component page on load.
+ *
+ * Renders main page consisting of FilterableDataTable, CitationPolicy
+ * and addAcknowledgement.
+ *
+ * @author Zaliqa Rosli
+ * @version 1.0.0
+ *
+ */
+class AcknowledgementsIndex extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      data: {},
+      formData: {},
+      error: false,
+      isLoaded: false,
+      affiliationsOptions: {
+        douglas: 'Douglas',
```

Comment:
Hard coding?

Reply:
yep, these were hardcoded in the module before reactification. the module doesn't currently have configurations options / tables in the back